compiler/hash-pest-parser/src/translate.rs

```diff
@@ -147,15 +141,17 @@ impl NodeBuilder {
         // we need to transform the lhs into a reference if the type of function is 'assigning'
         let lhs = self.transform_expr_into_ref(lhs, assiging);
 
-        let fn_call = self.node(Expression::FunctionCall(FunctionCallExpr {
-            subject: self.node(Expression::Variable(VariableExpr {
-                name: self.make_single_access_name(AstString::Borrowed("ord")),
-                type_args: vec![],
-            })),
-            args: self.node(FunctionCallArgs {
-                entries: vec![lhs, rhs],
-            }),
-        }));
+        let fn_call = self.node(Expression::new(ExpressionKind::FunctionCall(
+            FunctionCallExpr {
+                subject: self.node(Expression::new(ExpressionKind::Variable(VariableExpr {
+                    name: self.make_single_access_name(AstString::Borrowed("ord")),
```

Comment:
In the future, we should put common symbols into the hashmap before we run to avoid that additional first time adding cost.

Reply:
Yes, probably a file alongside `ident.rs` that injects them at startup.